templates/general/ssl.j2

```diff
@@ -8,4 +8,5 @@
   ssl_protocols {{ssl_protocols}};
   ssl_dhparam '{{ssl_dh_file}}';
   ssl_prefer_server_ciphers on;
-  ssl_session_cache shared:SSL:10m;
\ No newline at end of file
+  ssl_session_cache shared:SSL:10m;
+  add_header Strict-Transport-Security max-age=15768000;
```

Comment:
> The header should only be sent when encryption mode is force or redirect, but not optional. 

It's configured under the SSL server definition sooo...it's not always sent unless you're using SSL..